organizational-structure-overview.md

```diff
@@ -0,0 +1,86 @@
+# OpenSSF Governance & Org Structure
+
+## OpenSSF Charter
+
+The overall governance of the OpenSSF is defined in [the Foundation Charter](https://cdn.platform.linuxfoundation.org/agreements/openssf.pdf) (PDF file). 
+
+
+## Definitions
+
+The OpenSSF is comprised of instances of the following categories of official groups:
+
+- The **Governing Board** (GB) oversees the budget and legal status/structure of the OpenSSF. Board meetings may or may not be open to the public.
+- **Committees** are formed by, and report to, the Board. Broadly speaking, Committees handle non-technical matters as needed by the GB, wherein a named set of people are tasked with handling a specific objective. Like the Board meetings, Committee meetings may or may not be open to the public.
+- The **Technical Advisory Council** (TAC) is responsible for the general success of all Technical Initiatives (defined below). TAC meetings are generally open to the public, with the exception of special sessions. 
+  - TAC membership is limited to seven (7) seats, with alternating elections every year. Contributors to Technical Initiatves may run, and each contributor may vote once in TAC elections. 
+- **Technical Initiatives** are groups formed in support of the overall mission of the OpenSSF. Technical Initiatives include Working Groups (WG), Special Interest Groups (SIG), Projects, and services governed by their own Charters with various objectives, reporting relationships, and funding models described below.
+   - All Technical Initiatives are open groups focused on a technical objective, with transparent proceedings that are open to the public, and vary in their objectives.
+   - Technical Initiatives usually manage one or more GitHub repos of their own, may have separate meetings.
+- A **Project** is a Technical Initiative focused on the development and ongoing support of open source licensed software (source code) and its supporting artifacts (technical documentation, etc.).
+- A **Working Group** (WG) focuses on developing other types of deliverables than software such as guides, specifications, and educational material, or conducting initiatives such as an education outreach. While a WG does not produce open source software as a primary artifact it may launch Projects that do so. A WG may also launch Special Interest Groups (SIG) to perform specific tasks. WGs often include some open source code, or use licensed software, in fulfilment of their Charter.
+- **Special Interest Groups** (SIG) are under the direct governance of their reporting WGs and are bound to achieving a very specific goal. 
+- A **Technical Deliverable** can be an open source licensed software and its supporting artifacts, a specification, a technical guide. This is a technical artefact produced by a Technical Initiative.
```

Comment:
I would like to see in the charter a requirement that any "specification" be produced and licensed using the Community Specification license and process: https://github.com/CommunitySpecification/Community_Specification both for better clarity and so that anything moderately successful has an easier time making its way into formal standards efforts.

Reply:
(sorry, not in the charter, but in this document)

Reply:
> Suggest: "...a specification _or_ a technical guide".

> Also note: we have both US "artifact" and Commonwealth "artefact" spellings in the space of 2 sentences.

Addressed in PR#117

Reply:
>  any "specification" be produced and licensed using the Community Specification license and process: https://github.com/CommunitySpecification/Community_Specification

I think that's reasonable.

@brianbehlendorf do you have a suggested location for including that guidance? (It almost seems like a Charter thing, since the Charter covers acceptable Licenses already)

Reply:
I think this is actually already in the charter, in Exhibit B, Section 5.

Reply:
Yep, so having it here is just a reminder that it's that way in the charter.

Reply:
@brianbehlendorf - can you please suggest the exact change needed here, or lets do in future PR ?

Reply:
Lets do a future PR if called for after the dust settles on 112.